### PR TITLE
docs(nx-cloud): add feature page for GitHub integration

### PR DESCRIPTION
**Pull request by philipjfulcher**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/nx-cloud/features/github-integration.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Good CI checks require fast and easy access to results. That's why Nx Cloud will update your PR with the current running status of your tasks and a convenient link to your Nx Cloud results and logs. Take advantage of the enhanced developer experience of structured and searchable logs. Quick insight to PR task progress, so you're not stuck waiting for every task to complete. And with Nx Replay, developers can quickly replay tasks locally to avoid running tasks that CI has already completed.
+
+## Access control
+
+![Diagram showing users syncing from GitHub to Nx Cloud](/nx-cloud/features/github-user-management.avif)
+
+Nx Cloud organizations can use their existing GitHub access controls to manage Nx Cloud as well. This allows Nx Cloud to fit in to any existing on-boarding or off-boarding process. There's no need to manually manage users separately. Get your engineers Nx Cloud access right alongside their GitHub access so they can get to work fast. Use [personal access tokens](/ci/recipes/security/personal-access-tokens) to further enhance your security.
+
+## Getting started
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:23]:
```suggestion
## Get started
```

No "ing" in headers

File: docs/nx-cloud/features/github-integration.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+First, you'll want to connect your Nx Cloud account to GitHub. You can use your regular username and password or log in via Google or GitHub, connecting to GitHub is a separate step.
+
+{% call-to-action title="Log in to Nx Cloud" url="https://cloud.nx.app/profile/vcs-integrations" icon="nxcloud" description="Connect to GitHub via your profile settings" %}
+Connect to GitHub
+{% /call-to-action %}
+
+## Connecting a new workspace and organization
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:31]:
```suggestion
## Connect a new workspace and organization
```

File: docs/nx-cloud/features/github-integration.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+1. Visit [Nx Cloud](https://cloud.nx.app) and click **Connect a workspace** at the top.
+2. Follow the prompts to select a repo.
+3. If that repo is controlled by a GitHub organization, you will be prompted to use that organization.
+4. Follow the prompts to create a pull request to complete your connection to Nx Cloud.
+
+{% call-to-action title="Log in to Nx Cloud" url="https://cloud.nx.app/setup/connect-workspace/github/select" icon="nxcloud" description="Connect a new Nx workspace to Nx Cloud" %}
+Connect a new workspace to Nx Cloud
+{% /call-to-action %}
+
+## Connecting an existing organization
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:42]:
```suggestion
## Connect an existing organization
```

File: docs/nx-cloud/features/github-integration.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+If you've already created an organization in Nx Cloud, and you'd like to use your GitHub organization to manage access to it:
+
+1. Go to the organization in Nx Cloud while logged in as an admin user.
+2. Click on **Settings** in the top menu
+3. Go to **Connect GitHub organization in the sidebar**
+4. Follow the prompts there to connect to GitHub.
+
+## Connecting an existing workspace
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:51]:
```suggestion
## Connect an existing workspace
```

File: docs/nx-cloud/features/github-integration.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+Nx Cloud organizations can use their existing GitHub access controls to manage Nx Cloud as well. This allows Nx Cloud to fit in to any existing on-boarding or off-boarding process. There's no need to manually manage users separately. Get your engineers Nx Cloud access right alongside their GitHub access so they can get to work fast. Use [personal access tokens](/ci/recipes/security/personal-access-tokens) to further enhance your security.
+
+## Getting started
+
+First, you'll want to connect your Nx Cloud account to GitHub. You can use your regular username and password or log in via Google or GitHub, connecting to GitHub is a separate step.
+
+{% call-to-action title="Log in to Nx Cloud" url="https://cloud.nx.app/profile/vcs-integrations" icon="nxcloud" description="Connect to GitHub via your profile settings" %}
+Connect to GitHub
+{% /call-to-action %}
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:29]: This button's primary text is "Log in to Nx Cloud", but that isn't the main thing you're trying to get people to do.  Also, "Connect to GitHub" isn't displayed anywhere.  Maybe that should be the title?

File: docs/nx-cloud/features/github-integration.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+## Connecting a new workspace and organization
+
+1. Visit [Nx Cloud](https://cloud.nx.app) and click **Connect a workspace** at the top.
+2. Follow the prompts to select a repo.
+3. If that repo is controlled by a GitHub organization, you will be prompted to use that organization.
+4. Follow the prompts to create a pull request to complete your connection to Nx Cloud.
+
+{% call-to-action title="Log in to Nx Cloud" url="https://cloud.nx.app/setup/connect-workspace/github/select" icon="nxcloud" description="Connect a new Nx workspace to Nx Cloud" %}
+Connect a new workspace to Nx Cloud
+{% /call-to-action %}
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:40]: This button should not have the primary text of "Log in to Nx Cloud".  Maybe it should be "Connect a Workspace"?

File: docs/nx-cloud/features/github-integration.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+# GitHub integration
+
+Any CI tool requires tight integration with your existing version control system. Nx Cloud offers first class integration with GitHub in the following ways.
+
+## Easy Onboarding
+
+![Screenshot of Nx Cloud connecting a GitHub repository](/nx-cloud/features/github-onboarding.avif)
+
+Ready to get started with Nx Cloud? Get started with Nx Cloud in no time with our GitHub connection process. Connect your workspace, and Nx Cloud will create a pull request with everything you need. Select your workspace and organization, and Nx Cloud takes care of the rest. User access is automatically connected to GitHub, and a PR is created to connect your workspace. Your repo now has [distributed caching](/ci/features/remote-cache) in less than 5 minutes.
```
Comment (isaacplmann) [inline docs/nx-cloud/features/github-integration.md:9]:
```suggestion
Get started with Nx Cloud in no time with our GitHub connection process. Connect your workspace, and Nx Cloud will create a pull request with everything you need. Select your workspace and organization, and Nx Cloud takes care of the rest. User access is automatically connected to GitHub, and a PR is created to connect your workspace. Your repo now has [distributed caching](/ci/features/remote-cache) in less than 5 minutes.
```

Redundant first sentence

**Review by isaacplmann — CHANGES_REQUESTED** [review]

File: docs/nx-cloud/features/github-integration.md (hunk 6 above)
Comment (philipjfulcher) [reply docs/nx-cloud/features/github-integration.md]: Thanks, both CTAs have been updated with (hopefully) better copy

File: docs/nx-cloud/features/github-integration.md (hunk 5 above)
Comment (philipjfulcher) [reply docs/nx-cloud/features/github-integration.md]: Thanks, both CTAs have been updated with (hopefully) better copy

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**